scripts/run_multi_model.py

```
@@ -0,0 +1,341 @@
+    # Verify API config
+    if not args.dry_run:
+        if not os.getenv("CUSTOM_BASE_URL") or not os.getenv("CUSTOM_API_KEY"):
+            print("\nERROR: Set CUSTOM_BASE_URL and CUSTOM_API_KEY first.")
+            sys.exit(1)
+
+    reports = []
+    for model in args.models:
+        try:
+            report = run_model_tests(model, dry_run=args.dry_run)
+            reports.append(report)
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)